Include/datetime.h

```diff
@@ -155,12 +155,16 @@ typedef struct {
     PyTypeObject *DeltaType;
     PyTypeObject *TZInfoType;
 
+    /* singletons */
+    PyObject *TimeZone_UTC;
```

Comment:
Shouldn't we also add a macro definition below to simplify access to the singleton?

```
#ifdef Py_BUILD_CORE
 ..
#else
#define PyDateTime_TimeZone_UTC PyDateTimeAPI->TimeZone_UTC
#endif
```

The idea is to make code in user extensions look similar to that in _datetimemodule.c.

Reply:
Yeah, I'm fine with that. I think I didn't know of any examples of other singletons exposed in the C API so I didn't know to do this.